share/revng/test/tests/analysis/ImportFromCAnalysis/backward_compatible_padding_struct.model.yml.ccode

```diff
@@ -0,0 +1,9 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+struct _PACKED my_struct {
+  uint8_t _padding_at_0[8];
+  uint64_t *field;
+  uint8_t _padding_at_8[8];
+};
```

Comment:
I see that there is no REP for this PR. I'd like to know what are the design decisions about the following points:
1. Do we allow a mixed style of `_padding_at` and `_START_AT`?
2. If we don't allow mixed style, what happens at parse-time if the user uses mixed style?

Reply:
We really should make one 😅 

1. Yep. Everything but `_START_AT` attached to a `_padding_at` field is allowed. `_START_AT` takes precedence.

Reply:
By `_START_AT` takes precedence I mean that in cases like
```
struct whatever {
  uint8_t _padding_at_0[16];
  _START_AT(8)
  uint64_t field;
};
```

explicit padding is pretty much ignored.

Reply:
I don't think we have a test for the first thing you mentioned though, i.e. that `_START_AT` takes precedence. Can you please add it?

Also, I think that might be surprising, so make sure you `revng_log` it somewhere with message starting with `"WARNING: "`

Reply:
Done.